crates/bevy_ecs/src/change_detection.rs

```diff
@@ -199,3 +199,43 @@ pub struct ReflectMut<'a> {
 change_detection_impl!(ReflectMut<'a>, dyn Reflect,);
 #[cfg(feature = "bevy_reflect")]
 impl_into_inner!(ReflectMut<'a>, dyn Reflect,);
+
+pub struct MutUntyped<'a> {
```

Comment:
Doc comments <3 Link to `Mut`, and explain what this is for.

Reply:
I added docs everywhere :+1: